src/test/java/io/federecio/dropwizard/swagger/selenium/auth/DefaultServerWithAuthenticationSeleniumTest.java

```diff
@@ -17,6 +17,7 @@
 import java.util.List;
 import java.util.concurrent.TimeUnit;
 import org.junit.ClassRule;
+import org.junit.Ignore;
```

Comment:
I have used this link[1] to install the Chrome Driver and everything work prefectly. I have updated the release/0.9.x branch with the ChromeDriver tests also.

[1] https://christopher.su/2015/selenium-chromedriver-ubuntu/